hack/fetch-releases.sh

```diff
@@ -1,86 +1,113 @@
+     dir="trigger"
+    fi
+
+    #nightly -> 0.0.0-nightly
+    #latest -> find version till then -> 0.0.0-latest
+    #version -> directory with version
+
+    url=""
+    case $version in
+      nightly)
+        dirVersion="0.0.0-nightly"
+        url="https://storage.googleapis.com/tekton-releases-nightly/${comp}/latest/${releaseFileName}.yaml"
+        ;;
+      latest)
+        dirVersion="0.0.0-latest"
+        url="https://storage.googleapis.com/tekton-releases/${comp}/latest/${releaseFileName}.yaml"
+        ;;
+      *)
+        dirVersion=${version//v}
+        url="https://storage.googleapis.com/tekton-releases/${comp}/previous/${version}/${releaseFileName}.yaml"
+        ;;
+    esac
+
+    ko_data=${SCRIPT_DIR}/cmd/${TARGET}/operator/kodata
+    comp_dir=${ko_data}/tekton-${dir}
+
+    # before adding releases, remove existing version directories
+    # ignore while adding for interceptors
+    if [[ ${releaseFileName} != "interceptors.notags" ]] ; then
+      rm -rf ${comp_dir}/*
+    fi
+
+    # create a directory
+    dirPath=${comp_dir}/${dirVersion}
+    mkdir -p ${dirPath} || true
+
+    # destination file
+    dest=${dirPath}/${destFileName}.yaml
+    http_response=$(curl -s -o ${dest} -w "%{http_code}" ${url})
+    echo url: ${url}
+
+    if [ $http_response != "200" ]; then
+        echo "Error: failed to get $comp yaml, status code: $http_response"
+        exit 1
+    fi
+
+    # Add OpenShift specific files for pipelines
+    if [[ ${TARGET} == "openshift" ]] && [[ ${comp} == "pipeline" ]]; then
+      cp -r ${ko_data}/openshift/00-prereconcile ${comp_dir}/
+      cp ${ko_data}/openshift/pipelines-rbac/* ${dirPath}/
+    fi
+
+    if [[ ${comp} == "dashboard" ]]; then
+      sed -i '/aggregationRule/,+3d' ${dest}
+    fi
+
+    echo "Info: Added $comp/$releaseFileName:$version release yaml !!"
+    if [[ ${comp} == "results" ]]; then
+      grep 'version' ${dest} | head -n 1 | tr -d ' ' || true
+    else
+      grep 'app.kubernetes.io/version' ${dest} | head -n 1 | sed 's/[[:space:]]*app.kubernetes.io\///' || true
+    fi
+    echo ""
 }
 
-#Args: TARGET PIPELINES_VERSION TRIGGERS_VERSION
+#Args: <target-platform> <pipelines version> <triggers version> <dashboard version> <results version>
 main() {
   TARGET=$1
-
   p_version=${2}
-  release_yaml pipeline release 00-pipelines ${p_version}
-
   t_version=${3}
-  release_yaml triggers release 00-triggers ${t_version}
-  release_yaml triggers interceptors 01-interceptors ${t_version}
+  d_version=${4}
+  r_version=${5}
+
+  echo Target Platform: ${TARGET}
+  echo pipelines: ${p_version}
+  echo triggers: ${t_version}
+  echo dashboard: ${d_version}
+  echo results: ${r_version}
+  echo ""
+
+  release_yaml pipeline release.notags 00-pipelines ${p_version}
+
+  release_yaml triggers release.notags 00-triggers ${t_version}
+  release_yaml triggers interceptors.notags 01-interceptors ${t_version}
+
+  release_yaml dashboard tekton-dashboard-release 00-dashboard ${d_version}
```

Comment:
in case of openshift we don't need to fetch yaml for result and dashboard 
so probably we need to check here